code/modules/paperwork/photography.dm

```diff
@@ -199,7 +215,13 @@
 		atoms.Add(the_turf);
 		// As well as anything that isn't invisible.
 		for(var/atom/A in the_turf)
-			if(A.invisibility) continue
+			if(A.invisibility )
+				if(see_ghosts && istype(A,/mob/dead/observer))
+					var/mob/dead/observer/O = A

```

Comment:
okay so current train of though mimic the ghost sprite as
/////spooky helpers!/////
/obj/effect/spookyghost
	name = "Spooky Ghost"
	desc = "It's a g-g-g-g-ghooooost!" //jinkies!
	icon = 'icons/mob/mob.dmi'
	icon_state = "ghost"
	layer = 4
	density = 0
	anchored = 1	//  don't get pushed around

then try to set that when it calls the image...only i am doing it wrong and did a runtime on the layer bit...halp.